.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    'no-proto': 0
```

Comment:
The 'no-proto' rule is set to 0, which means it is turned off. Consider whether this is intentional, as using '__proto__' is generally discouraged due to performance and security reasons. If you need to use '__proto__', ensure that it is necessary and safe in your context.